lte/gateway/python/magma/enodebd/dp_client.py

```diff
@@ -45,3 +53,81 @@ def get_cbsd_state(request: CBSDRequest) -> CBSDStateResult:
         )
         return CBSDStateResult(radio_enabled=False)
     return res
+
+
+def _indoorTobool(s: str):
+    return s.lower() in ['true', 't', '1', 'yes', 'indoor']
+
+
+def build_enodebd_update_cbsd_request(
+    serial_number: str,
+    latitude_deg: str,
+    longitude_deg: str,
+    indoor_deployment: str,
+    antenna_height: str,
+    antenna_height_type: str,
+    antenna_gain: str,
+    cbsd_category: str,
+) -> EnodebdUpdateCbsdRequest:
+    # cbsd category and antenna height type should be converted to lowercase
+    # for the gRPC call
+    antenna_height_type = antenna_height_type.lower()
```

Comment:
if zero value for antenna_height_type is ' ' than ignore this comment. If it's None however, this will crash.

Reply:
In this step, the parameters must be set in the device configs, so if there is "no value" for a given parameter, it will be an empty string.